README.md

```diff
@@ -85,6 +86,26 @@ The following rules can be configured per language. Add a `<language>.toml` file
 | quote_start_with_letter |  If a quote needs to start with a letter | boolean | true
 | replacements |  Replaces abbreviations or other words according to configuration | Array of replacement configurations: each configuration is an Array of two values: `["search", "replacement"]`. See example below. | nothing gets replaced
 
+### Example for `matching_symbols`
+
+```
+matching_symbols = [
+  ["„", "“"],
+  ["(", "]"],
+  ["[", ")"],
+]
+```
+
+This matches all occurrence of `„` with `“`, all occurrence of `(` with `]`, all occurrence of `[` with `)`.
+
+```
+Input: This is „a test“ and (another one]
+Output: Valid
```

Comment:
I see how this could be useful to show that it doesn't need to be semantically the opposite character. However, on the first look this might be confusing. Maybe we could use ( and ), [ and ] for the example and add a second one which explains that what you did here is in theory possible as well?